.changeset/cool-bikes-kick.md

```diff
@@ -0,0 +1,2 @@
+---
```

Comment:
❓ If this PR affects things already in production, I think we should add a changeset. :)

Reply:
Thanks for the reminder, no this is not part of release flow yet. 

When domains was added for the first time [there was a chanset](https://github.com/clerkinc/javascript/blob/3dc622a6b97436e2265eaacf29ebfeb1a72da9ef/.changeset/nervous-lizards-repeat.md) describing what new was added in the OrganizationProfile component.